open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/SubjectAreaClient.java

```diff
@@ -9,65 +9,190 @@
 
 import java.util.List;
 
+/**
+ * @param <T> object type for supplied and return.
+ * Interface describing common methods to client working with Subject area resources.
+ */
 public interface SubjectAreaClient<T> {
 
     FindRequest EMPTY_FIND_REQUEST = new FindRequest();
 
+    /**
+     * Get object by guid
+     *
+     * @param userId unique identifier for requesting user, under which the request is performed.
+     * @param guid   unique identifier of the object to which the found objects should relate.
+     * @return found objects of the T type.
+     *
+     * @throws PropertyServerException    something went wrong with the REST call stack.
+     * @throws UserNotAuthorizedException the requesting user is not authorized to issue this request.
+     * @throws InvalidParameterException  one of the parameters is null or invalid.
+     */
     T getByGUID(String userId, String guid) throws InvalidParameterException,
                                                    PropertyServerException,
                                                    UserNotAuthorizedException;
 
+    /**
+     * Create a object. To create, you must pass the created object and specify a unique user identifier.
```

Comment:
object => node

Reply:
by this do you mean a unique user identifier is the qualified name. This was optional and if not specified we talked of defaulting it. I assume you are not talking about creating a node specifying a guid, this is definitely not required.

Reply:
Analogically.  In this case, there may be other objects (relationship for example and etc) and not only node.